src/loadlib.js

```diff
@@ -95,6 +113,17 @@ if (!WEB) {
 
         return true;
     };
+} else {
+    /* TODO: use async/await ? */
+    readable = function(filename) {
+        let xhr = new XMLHttpRequest();
+        /* Following GET request done by searcher_Web will be cached */
+        xhr.open("GET", lua.to_jsstring(filename), false);
```

Comment:
If we do the browser will effectively do 2 request (HEAD in readable, and then GET to actually get the file). If we do a GET right away, the following request will use the cache.